models/ed/R/model2netcdf.ED2.R

```diff
@@ -964,7 +1004,11 @@ put_E_values <- function(yr, nc_var, out, lat, lon, begins, ends, pft_names, ...
   # so that read.output has a way of accessing PFT names
   nc_var[[s+4]]<- ncdf4::ncvar_def("PFT", units = "", dim = list(p),  
                                    longname = paste(pft_names, collapse=",")) 
-  
+  out_length <- length(out)
+  out[[out_length+1]] <- c(rbind(bounds[,1], bounds[,2]))
+  nc_var[[s+5]] <- ncdf4::ncvar_def(name="dtime_bounds", units='', 
+                                                 longname = "monthly history time interval endpoints", dim=list(time_interval,dtime = t), 
+                                                 prec = "double")
```

Comment:
Can these be changed to access `nc_var` by name instead of `s + 4` and `s + 5`? If so, I would _strongly_ encourage that -- right now, 4 and 5 look like magic numbers.

Reply:
That logic was implemented previously and I would have to look more deeply into why @istfer did it that way....there is probably a very good reason for it.

Reply:
If there is a good reason, it should be documented here. We'll need @istfer to weigh in.

Reply:
Based on this and the issue #2102 can we merge and strive toward generalizing as @ashiklom requested?

Reply:
I'm OK with that.